Cargo.toml

```diff
@@ -36,6 +36,7 @@ sic_parser = { version = "0.12.0", path = "components/sic_parser"}
 
 anyhow = "1.0.31"
 atty = "0.2.14"
+boolean = "0.3.0"
```

Comment:
I assume this crate was used because it handles case insensitivity in its `FromStr` implementation, which might be convenient with Claps `value_t`. So far I didn't use `value_t` yet but I can see why it might be convenient. 

I originally wrote the boolean library just to have a target to test my parameterized crate on.

Anyways, I would also be fine with including a boolean wrapper which implements FromStr case insensitively directly, if you think case insensitivity would be useful, you call :)! e.g. like so:
```
struct Boolean(bool); 

impl FromStr for Boolean {
  type Err = (some err type)
   
  fn from_str(from: &str) -> Result<Self, Self::Err> {
    match from.to_lowercase().as_str() {
      "true" => Ok(Boolean(true))
      "false" => Ok(Boolean(false))
      _ => Err( ... some err )
    }
  }
}
```